testspec.yml

```diff
@@ -27,6 +27,8 @@ phases:
   post_build:
     commands:
       - python src/send_status.py --status $CODEBUILD_BUILD_SUCCEEDING
+      # TODO: Enable test notification once ready
```

Comment:
Will create a separate PR to enable uncomment this, as a mark to making the feature live